examples/styling/draw-order.html

```diff
@@ -14,22 +14,40 @@
 </head>
 <body>
   <div id="map"></div>
-  <div id="controls">
-    <ul>
-      <li>
-        <input type="radio" name="style" onclick="setDefaultOrder()" id="default" checked>
-        <label for="default">Default</label>
-      </li>
-      <li>
-        <input type="radio" name="style" onclick="setAscOrder()" id="asc">
-        <label for="asc">Ascending</label>
-      </li>
-      <li>
-        <input type="radio" name="style" onclick="setDescOrder()" id="des">
-        <label for="des">Descending</label>
-      </li>
-    </ul>
-  </div>
+    <aside class="toolbox">
+      <div class="box">
+        <header>
+          <h1>Draw Order</h1>
+        </header>
+        <section>
+          <p class="description open-sans">How to use CARTO ordering style property</p>
+          <div id="controls">
+            <ul>
+              <li>
+                <input type="radio" name="style" onclick="setAscOrder()" checked>
```

Comment:
Please keep the `id` to allow changing the radio button by clicking the text (that's why the for="" exists)